common.js

```diff
@@ -1,10 +1,13 @@
+'use strict'
+
```

Comment:
Well, since electron-packager requires nodejs 4.0, we just can use it (i.e. nothing to discuss here), if you insist, I can remove it.